test/runtests.jl

```diff
@@ -1,9 +1,15 @@
-using NanoporeMultiEventalign
 using Test
+using NanoporeMultiEventalign
 
 @testset "dtw.jl" begin
     @testset "dtw" begin
         @test dtw(true)
         @test !dtw(false)
     end
 end
+
+@testset "persistence.jl" begin
+    @testset "loadReference" begin
+        @test convert(String,loadReference("sample_data/reference.fa")) == "GAATACAAGCTACTTGTTCTTTTTGCAGGATCCCATCGATTCGAATTCAAGGCCTCTCGAGCCTCTAGAACTATAGTGAGTCGTATTACGTAGATCCAGACATGATAAGATACATTGATGAGTTTGGACAAACCACAACTAGAATGCAGTGAAAAAAATGCTTTATTTGTGAAATTTGTGATGCTATTGCTTTATTTGTAACCATTATAAGCTGCAATAAACAAGTTAACAACAACAATTGCATTCATTTTATGTTTCAGGTTCAGGGGGAGGTGTGGGAGGTTTTTTAATTCGCAAAAAAAAAAGGGAGAGCGACGAGAAGAAGAGGCTTGTATGGACGATGTCTTCCACAACAGAAGACATCGTCCATACAAGCAAAAAA"
```

Comment:
split the string into multiple lines and concatenate with *, sot hat it does not look bad